src/crypto/ctaes/COPYING

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2016 Pieter Wuille
```

Comment:
This needs an entry in `contrib/debian/copyright`.

Reply:
I had started to do this, and then saw that `contrib/debian/copyright` did not contain any MIT-licensed code (such as `src/secp256k1` which is similarly a git subtree of a separate repository) and assumed that this was intentional.

Reply:
That is strictly speaking an error that should also be fixed. src/secp256k1/* is "Copyright (c) 2013 Peter Wiulle", which differs from the overall default license which is "Copyright: 2016-2020, The Zcash developers; 2009-2020, Bitcoin Core developers; 2009-2020, Bitcoin Developers".

(Yes, Peter Wiulle happens to be a Bitcoin Core developer. But these two packages *aren't* copyright by other Zcash/Bitcoin Core/Bitcoin developers, and that should be recorded.)